crates/fuel-streams-core/src/outputs/subjects.rs

```diff
@@ -0,0 +1,143 @@
+use fuel_core_types::{fuel_tx::TxId, fuel_types::ContractId};
+use fuel_streams_macros::subject::{IntoSubject, Subject};
+
+use crate::types::*;
+
+pub const OUTPUTS_WILDCARD_LIST: &[&str] = &[
```

Comment:
Since the only place you have this being consumed is the `impl` in the `mod.rs` you don't need to export this here, you can pass directly into the implementation (as it's in the other places)